sdk/appconfiguration/azure-appconfiguration/tests/test_azure_appconfiguration_client_async.py

```diff
@@ -1005,6 +1045,122 @@ async def test_list_snapshot_configuration_settings(self, appconfiguration_conne
 
         await self.tear_down()
 
+    @app_config_decorator_async
+    @recorded_by_proxy_async
+    async def test_monitor_configuration_settings_by_page_etag(self, appconfiguration_connection_string):
+        # response header <x-ms-content-sha256> and <x-ms-date> are missing in python38.
+        set_custom_default_matcher(compare_bodies=False, excluded_headers="x-ms-content-sha256,x-ms-date")
+        async with AzureAppConfigurationClient.from_connection_string(appconfiguration_connection_string) as client:
+            # prepare 200 configuration settings
+            for i in range(200):
+                await client.add_configuration_setting(
+                    ConfigurationSetting(
+                        key=f"async_sample_key_{str(i)}",
+                        label=f"async_sample_label_{str(i)}",
+                    )
+                )
+            # there will have 2 pages while listing, there are 100 configuration settings per page.
+
+            # get page etags
+            page_etags = []
+            items = client.list_configuration_settings(
+                key_filter="async_sample_key_*", label_filter="async_sample_label_*"
+            )
+            iterator = items.by_page()
+            async for page in iterator:
+                etag = iterator.etag
+                page_etags.append(etag)
+
+            # monitor page updates without changes
+            continuation_token = None
+            index = 0
+            request = HttpRequest(
+                method="GET",
+                url="/kv?key=async_sample_key_%2A&label=async_sample_label_%2A&api-version=2023-10-01",
+                headers={
+                    "If-None-Match": page_etags[index],
+                    "Accept": "application/vnd.microsoft.appconfig.kvset+json, application/problem+json",
+                },
+            )
+            first_page_response = await client.send_request(request)
+            assert first_page_response.status_code == 304
+
+            link = first_page_response.headers.get("Link", None)
+            continuation_token = link[1 : link.index(">")] if link else None
+            index += 1
+            while continuation_token:
+                request = HttpRequest(
+                    method="GET", url=f"{continuation_token}", headers={"If-None-Match": page_etags[index]}
```

Comment:
Is there no way to do this in the SDK?

Reply:
@mrm9084 : Commenting on a PR that was closed out 8 months ago is probably not the most efficient path for a question.   I'd suggest starting an email thread and adding detail around the end-to-end scenario that you're interested in.

Reply:
I was using this a link to the line that I was looking at to send to Yalin.

Reply:
This feature is in all of the other SDKs, I'm trying to figure out why not python.